cmd/expiration-mailer/main.go

```diff
@@ -224,7 +224,7 @@ func (m *mailer) sendNags(conn bmail.Conn, contacts []string, certs []*x509.Cert
 		TruncatedDNSNames  string
 		NumDNSNamesOmitted int
 	}{
-		ExpirationDate:     expDate.UTC().Format(time.RFC822Z),
+		ExpirationDate:     expDate.UTC().Format("2006-01-02"),
```

Comment:
Consider making this format a constant.

Reply:
This is available as a constant as `time.DateOnly`

Reply:
Ah, thanks for pointing that out.